File: multi-db/src/main/java/com/multidb/repository/employee/entity/SalariesEntity.java
package com.multidb.repository.employee.entity;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.time.LocalDate;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@JsonInclude(JsonInclude.Include.NON_NULL)
@Entity
@Table(name = "salaries")
public class SalariesEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @Column(name = "emp_no")
    private Integer empNo;

    @Column(name = "salary")
    private Integer salary;

    @Column(name = "from_date")
    private LocalDate fromDate;

    @Column(name = "to_date")
    private LocalDate toDate;

//    @Column(name = "gender")
//    @Enumerated(EnumType.ORDINAL)
//    var gender: Gender? = null,

    //    @Enumerated(EnumType.ORDINAL)
    //    @Column(name = "gender")
    //    private Gender gender;
    @Transient
    @JsonProperty
    private Integer pageSize;

    @Transient
    @JsonProperty
    private Integer currentPage;
}

File: multidatasource/src/main/java/com/multidatasource/repository/employee/entity/SalariesEntity.kt
package com.multidatasource.repository.employee.entity

import com.fasterxml.jackson.annotation.JsonInclude
import com.fasterxml.jackson.annotation.JsonProperty
import java.io.Serializable
import java.time.LocalDate
import javax.persistence.*

@Entity
@Table(name = "salaries")
@JsonInclude(JsonInclude.Include.NON_NULL)
open class SalariesEntity (
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @Column(name = "emp_no")
    open var empNo: Int? = 0,

    @Column(name = "salary")
    open var salary: Int? = null,

    @Column(name = "from_date")
    open var fromDate: LocalDate? = null,

    @Column(name = "to_date")
    open var toDate: LocalDate? = null,

//    @Column(name = "gender")
//    @Enumerated(EnumType.ORDINAL)
//    var gender: Gender? = null,

    //    @Enumerated(EnumType.ORDINAL)
    //    @Column(name = "gender")
    //    private Gender gender;
    @Transient
    @JsonProperty
    open var pageSize: Int? = null,

    @Transient
    @JsonProperty
    open var currentPage: Int? = null

) : Serializable